### PR TITLE
Update HighPerformanceComputing.md

### PR DESCRIPTION
**Pull request by mattfidler**
Add parallel ode solving and nonlinear mixed effects.

Note that `rxode2` and other ODE solutions (like `mrgsolve`) compile code for faster solving, but I thought it was a bit out of scope for the section (feel free to agree/disagree here)

### CODE REVIEW

**Comment by eddelbuettel** [comment]
The meta question is whether this is HPC simply because OpenMP is involved.   On the margin "I guess so" as that criterion was used previously too, seemingly, but it is getting murkier.

Your use of sitmo is internal (i.e. "vendored") and does not use the package [sitmp](https://cran.r-project.org/package=sitmo) or does it?

**Comment by mattfidler** [comment]
> The meta question is whether this is HPC simply because OpenMP is involved. On the margin "I guess so" as that criterion was used previously too, seemingly, but it is getting murkier.

This is a larger question (one I can't answer).

Yet, as far as I know it is the only R-based ODE solver that does thread-based solving. But is this high-performance computing, not sure.

> Your use of sitmo is internal (i.e. "vendored") and does not use the package [sitmp](https://cran.r-project.org/package=sitmo) or does it?


The package is not vendored, but extends `sitmo` to simulate various distributions that can be done in parallel.  The technical package (that can be called in R) is `rxode2random`.  Hence you can use that to run multivariate normal simulations in parallel or simply binomial simulations in parallel.  I thought to simplify it by calling out the `rxode2` interface.  Originally it was only in `rxode2` but CRAN asked us to reduce its compilation time and we split it out.

**Comment by eddelbuettel** [comment]
Yes. I found the sitmo to rxode2random link too by looking at the package page.  Is that something for your proposed paragraphs should have an appropriate hyperlink?

**Comment by mattfidler** [comment]
I guess it does make sense if people are using `mvnfast` only that they may wish to use `rxode2random` for truncated multivariate normal without the overhead of `rxode2` (which I don't believe is available in `mvnfast`).  I added some text for that too.

**Comment by mattfidler** [comment]
The `rxode2` paragraph is getting a bit wordy, it could be split into 2 calling out the `rxode2random` first maybe...

**Comment by mattfidler** [comment]
Updated with a split paragraph for `rxode2random`

**Review by eddelbuettel — COMMENTED** [review]
Looks good to me -- let's roll.

**Review by eddelbuettel — APPROVED** [review]

**Comment by mattfidler** [comment]
Thanks!
